patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1504,12 +1504,24 @@
 +        return isBurning(world, pos) ? net.minecraft.pathfinding.PathNodeType.DAMAGE_FIRE : null;
 +    }
 +
++    /**
++     * Determines if a block will allow snow to be placed on it, this allows the implement to
++     * change the value to true or false overriding the behavior entirely.
++     * 
++     * @param the default return value, based on solid sides and isLeaves and other vanilla checks.
++     * @return true will allow snow to be placed, false will prevent it from being placed.
++     */
++    public boolean canPlaceSnowOnTop(boolean vanillaDefaultBehavior)
```

Comment:
More arguments?

Reply:
Sorry? I'm not sure what you mean?

Reply:
`public boolean canPlaceSnowOnTop(IBlockState state, boolean vanillaDefaultBehavior)`

Reply:
Wow... I really dropped the ball on that didn't I...